17/bullseye/Dockerfile

```
@@ -34,7 +41,7 @@ RUN jlink \
          --compress=2 \
          --output /javaruntime
 
-FROM debian:bullseye-20221024
+FROM debian:bullseye-20221024 as build
```

```suggestion
FROM debian:bullseye-20221024 AS build
```

I'm being nit picky.